src/vcpkg/postbuildlint.cpp

```diff
@@ -905,38 +1065,25 @@ namespace vcpkg::PostBuildLint
         OutdatedDynamicCrt outdated_crt;
     };
 
-    static LintStatus check_outdated_crt_linkage_of_dlls(const std::vector<Path>& dlls,
-                                                         const Path& dumpbin_exe,
+    static LintStatus check_outdated_crt_linkage_of_dlls(const std::vector<PostBuildCheckDllData>& dlls,
                                                          const BuildInfo& build_info,
                                                          const PreBuildInfo& pre_build_info)
     {
         if (build_info.policies.is_enabled(BuildPolicy::ALLOW_OBSOLETE_MSVCRT)) return LintStatus::SUCCESS;
 
+        const auto outdated_crts = get_outdated_dynamic_crts(pre_build_info.platform_toolset);
         std::vector<OutdatedDynamicCrtAndFile> dlls_with_outdated_crt;
 
-        for (const Path& dll : dlls)
+        for (const PostBuildCheckDllData& dll_data : dlls)
         {
-            auto cmd_line = Command(dumpbin_exe).string_arg("/dependents").string_arg(dll);
-            const auto maybe_output = flatten_out(cmd_execute_and_capture_output(cmd_line), "dumpbin");
-            if (const auto output = maybe_output.get())
+            for (const OutdatedDynamicCrt& outdated_crt : outdated_crts)
             {
-                for (const OutdatedDynamicCrt& outdated_crt :
-                     get_outdated_dynamic_crts(pre_build_info.platform_toolset))
+                if (Util::Vectors::contains(dll_data.dependencies, outdated_crt.name))
```

Comment:
This was a case-insensitive compare before -- is that preserved / no longer needed?

Reply:
Yes, see 982 above:

```
            for (auto&& directive : lib_info.linker_directives)
            {
                if (Strings::case_insensitive_ascii_equals(directive, static_release_crt))
                {
                    this_lib.has_static_release = true;
                }
                else if (Strings::case_insensitive_ascii_equals(directive, static_debug_crt))
                {
                    this_lib.has_static_debug = true;
                }
                else if (Strings::case_insensitive_ascii_equals(directive, dynamic_release_crt))
                {
                    this_lib.has_dynamic_release = true;
                }
                else if (Strings::case_insensitive_ascii_equals(directive, dynamic_debug_crt))
                {
                    this_lib.has_dynamic_debug = true;
                }
            }
```

Reply:
Fixed!